src/app/authentication/commons/constants/constants.go

```diff
@@ -6,6 +6,11 @@ const (
 	PortDefaultValue = 8080
 )
 
+// success messages of otp validation
+const (
+	ValidateOTPSuccessMessage = "OTP validated successfully"
```

Comment:
add the constant in same group 
all the successMessages of all APIs will be in same group
also change comment to // authentication success messages

Reply:
Resolved